django_meili/management/commands/syncindex.py

```diff
@@ -11,19 +11,38 @@
 from django_meili._client import client as _client
 from django_meili.models import IndexMixin
 
+
+def batch_qs(qs, batch_size=1000):
+    """
+    Returns a (start, end, total, queryset) tuple for each batch in the given
+    queryset.
+    
+    Usage:
+        # Make sure to order your querset
+        article_qs = Article.objects.order_by('id')
+        for start, end, total, qs in batch_qs(article_qs):
+            print "Now processing %s - %s of %s" % (start + 1, end, total)
+            for article in qs:
+                print article.body
+    """
+    total = qs.count()
+    for start in range(0, total, batch_size):
+        end = min(start + batch_size, total)
+        yield (start, end, total, qs[start:end])
```

Comment:
If it doesn't need to, I don't want to yield all of this. Can we have it just yield the batched queryset.

Reply:
yep fair enough

Reply:
✅